.travis.yml

```diff
@@ -68,7 +68,7 @@ jobs:
     - stage: "Analysis"
       name: "phpstan"
       script:
-        - docker-compose run --no-deps --rm php bin/phpstan analyze -l7 src/ tests/
+        - docker-compose run --no-deps --rm php bin/phpstan analyze --memory-limit -l7 src/ tests/
```

Comment:
`--memory-limit` requires an argument. What is its purpose here?

Reply:
I didn't know it required it! It probably has a `-1` default, because putting it there fixed the build.